elasticassignment/ageofempires/Elastic_search/elastic_client.py

```diff
@@ -0,0 +1,12 @@
+import requests
+import elasticsearch
+from django.conf import settings
+
+class elastic_client:
```

Comment:
- correct the name: `ElasticClient`
- Please make `make_connection` method private.
- add `__init__` function and instantiate `self.__connection`.
- correct the functionality of `get_data` function. It should retrieve data from Elastic indexes.
- add `index_data` function.
- add `create_index` function, it should accept `settings` and `mappings` as params.
- add any other function that needs direct connection with Elasticsearch.